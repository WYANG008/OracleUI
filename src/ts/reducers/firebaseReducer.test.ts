import { firebaseReducer, initialState } from './firebaseReducer';

describe('firebase reducer', () => {
	let state = initialState;

	test('default', () => {
		state = firebaseReducer(state, { type: 'any' });
		expect(state).toMatchSnapshot();
	});
});
