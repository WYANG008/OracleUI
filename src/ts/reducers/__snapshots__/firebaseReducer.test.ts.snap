// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`firebase reducer default 1`] = `
Object {
  "auth": false,
}
`;
